zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm

```diff
@@ -0,0 +1,126 @@
+/mob/living/carbon/superior_animal/roach/kaiser/kaiserin
+	name = "Kaiserin Roach"
+	desc = "The empress her self. A weapon to surpass metal gear."
+	icon = 'zzzz_modular_occulus/icons/mob/kaiserin.dmi'
+	icon_state = "kaiserin_roach"
+	icon_living = "kaiserin_roach"
+	icon_dead = "kaiserin_roach_dead"
+	density = TRUE
+	spawn_blacklisted = TRUE
+	rarity_value = 100
+	maxHealth = 2000
+	health = 2000
+	mob_size = MOB_HUGE
+	pixel_x = -16
+	ranged = TRUE
+	move_to_delay = 12
+	break_stuff_probability = 95
+	stop_automated_movement = 1
+
+	melee_damage_lower = 30
+	melee_damage_upper = 40
+
+
+	wander = FALSE //No more sleepwalking
+	projectilesound = 'zzzz_modular_occulus/sound/weapons/guns/fire/minigun_fire.ogg'
+	projectiletype = /obj/item/projectile/bullet/srifle/nomuzzle
+
+
+
+/mob/living/carbon/superior_animal/roach/kaiser/kaiserin/handle_ai()
```

Comment:
Why is this being replicated? If it is identical to the Kaiser proc, we don't need to re-define it

Reply:
...Ah- Yeah. Originally I had it's minigun shooting function from there, but I removed it in favor of other code. Thats my bad.